wolfcrypt/src/port/atmel/README.md

```diff
@@ -96,5 +96,43 @@ ATECC508A HW accelerated implementation:
 `EC-DSA   sign   time     293.400 milliseconds, avg over 5 iterations, 17.065 ops/sec`
 `EC-DSA   verify time     208.400 milliseconds, avg over 5 iterations, 24.038 ops/sec`
 
+### Microchip Trust Anchor TA100 ECC
+
+`./configure CFLAGS="-DWOLFSSL_CMAC -DHAVE_PK_CALLBACKS -DWOLFSSL_ATECC508A_NOIDLE -DECC_USER_CURVES -DWOLFSSL_ATECC_NO_ECDH_ENC -DWOLFSSL_ATECC_DEBUG" --enable-cmac --enable-microchip=100 --with-cryptoauthlib`
+
+
+```
+ $ lscpu -e
+CPU SOCKET CORE L1d:L1i:L2 ONLINE    MAXMHZ   MINMHZ
+  0      0    0 0:0:0         yes 1800.0000 600.0000
+  1      0    1 1:1:0         yes 1800.0000 600.0000
+  2      0    2 2:2:0         yes 1800.0000 600.0000
+  3      0    3 3:3:0         yes 1800.0000 600.0000
+
+$ uname -a
+Linux raspberrypi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux
+
+Software:
+------------------------------------------------------------------------------
+ wolfSSL version 5.6.0
+------------------------------------------------------------------------------
+Math:     Multi-Precision: Wolf(SP) word-size=64 bits=4096 sp_int.c
+wolfCrypt Benchmark (block bytes 1048576, min 1.0 sec each)
+
+ECC   [      SECP256R1]   256  key gen       700 ops took 1.065 sec, avg 1.522 ms, 657.067 ops/sec
```

Comment:
Where is RSA? I see you added support for it, so thought it would be listed here too?

Reply:
RSA and AES-GCM numbers would be nice to have.